zoo/policies/open-agent/open_agent/agent.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
 from typing import Sequence
 
+# pytype: disable=import-error
```

Comment:
```suggestion
```

Reply:
`casadi` or `opengen` will fail the CI test because they are not installed in the other packages